src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -504,7 +520,7 @@ public virtual bool IsEcsExcemptedLambda(int index)
             return false;
         }
 
-        public virtual bool AllowsRowScopedParamDelegationExempted(int index)
+        public virtual bool AllowsRowScopedParamDelegationExempted(int index, Features features)
```

Comment:
>, Features features [](http://example.com/codeflow?start=76&length=19)

Can we avoid this? 
has higher impact, and breaking (new parameter on method).

Reply:
We use the flag in AddColumns.cs, in public override bool AllowsRowScopedParamDelegationExempted(int index, Features features)

Reply:
Found an alternative